lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -64,19 +65,21 @@ class LinkBlockingFirstPaintAudit extends Audit {
     const results = filtered.map(item => {
       endTime = Math.max(item.endTime, endTime);
 
+      const transferSizeKbs = Util.formateBytesToKB(Math.round(item.transferSize));
+
       return {
         url: item.tag.url,
-        totalKb: `${Math.round(item.transferSize / 1024)} KB`,
-        totalMs: `${Math.round((item.endTime - startTime) * 1000)}ms`
+        totalKb: `${transferSizeKbs} KB`,
```

Comment:
could do this inline and remove the `Math.round` for just `formatBytesToKB(item.transferSize, 0)`